std/meta.d

```diff
@@ -986,24 +985,23 @@ template aliasSeqOf(alias range)
         {
             alias aliasSeqOf = AliasSeq!();
         }
-        else static if (range.length == 1)
-        {
-            alias aliasSeqOf = AliasSeq!(range[0]);
-        }
         else
         {
-            alias aliasSeqOf = AliasSeq!(aliasSeqOf!(range[0 .. $/2]), aliasSeqOf!(range[$/2 .. $]));
+            alias aliasSeqOf = AliasSeq!(range[0], aliasSeqOf!(range[1 .. $]));
```

Comment:
I have just committed after trying to squash:

`git rebase -i HEAD~2`

don't know if it worked. I tried again with:

`git rebase -i HEAD~4`

I hope it didn't mess anything up. My git skills leave something to be desired.